rp2040-hal/src/timer.rs

```diff
@@ -1,8 +1,8 @@
 //! Timer Peripheral
 //!
-//! The Timer peripheral on RP2040 consists of a 64-bit counter and 4 alarms.  
-//! The Counter is incremented once per microsecond. It obtains its clock source from the watchdog peripheral, you must enable the watchdog before using this peripheral.  
-//! Since it would take thousands of years for this counter to overflow you do not need to write logic for dealing with this if using get_counter.  
+//! The Timer peripheral on RP2040 consists of a 64-bit counter and 4 alarms.
+//! The Counter is incremented once per microsecond. It obtains its clock source from the watchdog peripheral, you must enable the watchdog before using this peripheral.
+//! Since it would take thousands of years for this counter to overflow you do not need to write logic for dealing with this if using get_counter.
```

Comment:
Please leave these spaces. In Markdown they force line feed within a paragraph (`<br/>`).

Reply:
Well, this should be happened due to automatic trailing white space removal of the editor. Will revert it back to the original.

Reply:
Done.